### ISSUE TITLE
Bug - stream shift error

### ISSUE THREAD

**Issue by leoraba** [cancogen-virus-seq/pedigree#36]
# Description
Pedigree stops running after some time(~1h 30mins) it throws the following error:
```
app/node_modules/stream-shift/index.js:16
    return state.buffer[0].length
                           ^

TypeError: Cannot read properties of null (reading 'length')
    at getStateLength (/app/node_modules/stream-shift/index.js:16:28)
    at shift (/app/node_modules/stream-shift/index.js:6:99)
    at Duplexify._forward (/app/node_modules/duplexify/index.js:170:35)
    at PassThrough.onreadable (/app/node_modules/duplexify/index.js:136:10)
    at PassThrough.emit (node:events:518:28)
    at emitReadable_ (node:internal/streams/readable:834:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:89:21)

Node.js v22.11.0
```

# Insight
* Error start happing after a Node.js update, from node 16 to node 22.

**Comment by leoraba** [comment]
issue related to [stream-shift](https://github.com/mafintosh/stream-shift/issues/6)

**Comment by leoraba** [comment]
Issue resolved in Release version 1.0.5 🎉
